walkthroughs/howto-http2/mesh/route-red-blue-green.json

```diff
@@ -19,7 +19,7 @@
                 ]
             },
             "match": {
-		"prefix": "/"
+		            "prefix": "/"
```

Comment:
Any reason for the whitespace changes in the route jsons?

Reply:
Hm well the spaces were inconsistent when I opened these files locally (so I fixed them) but on GitHub it looks like I made them worse.